app/main/views/assessments.py

```diff
@@ -92,3 +93,31 @@ def reject_assessment(id):
     except IntegrityError:
         abort(400)
     return jsonify(assessment.serializable), 200
+
+
+@main.route('/assessments/supplier', methods=['POST'])
+def get_supplier_assessments():
```

Comment:
not a fan of posting with id in body, why not `/assessments/supplier/<int:supplier_code>`